front/src/components/CompetitionList/CompetitionList.css

```diff
@@ -0,0 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@600;800&display=swap");
+
+.competitionsDataList {
+    font-family: "Raleway", sans-serif;
+    color: #8a7c58;
+}
+
+#mainHeader {
+    font-size: 32px;
+    text-decoration: none;
+    text-decoration: underline;
+    text-align: center;
+    margin-top: 60px;
+    margin-bottom: 15px;
+    font-weight: 900;
+}
+
+.table {
+    margin: 0 auto;
+}
+.table {
+    table-layout: auto !important;
+}
+.table tbody tr:hover {
+    /* font-weight: 700 !important; */
+    background-color: rgba(110, 112, 10, 0.12) !important;
```

Comment:
Please, define common palette in :root node and use it across all .css files. The faster we begin, the better.